pkgs/top-level/aliases.nix

```diff
@@ -564,6 +564,7 @@ mapAliases {
   EmptyEpsilon = empty-epsilon; # Added 2024-07-14
   enyo-doom = enyo-launcher; # Added 2022-09-09
   epdfview = throw "'epdfview' has been removed due to lack of maintenance upstream. Consider using 'qpdfview' instead"; # Added 2024-10-19
+  ephemeral = throw "'ephemeral' has been archived upstream 2022-04-02"; # added 2025-04-12
```

Comment:
nit

```suggestion
  ephemeral = throw "'ephemeral' has been archived by upstream since 2022-04-02"; # added 2025-04-12
```

Reply:
https://github.com/NixOS/nixpkgs/blob/4a610ef5776b37616429a7c9f13666f1f65d5387/pkgs/top-level/aliases.nix#L737
I see no precedent for "archived by upstream", and many cases of "archived upstream"

"since" is a valid suggestion.